src/tools/rustdoc-js/tester.js

```diff
@@ -289,7 +297,11 @@ function main(argv) {
                 }
             }
         }
-        if (error_text.length !== 0) {
+        if (should_fail_happy === false && should_fail === true) {
+            errors += 1;
+            console.error("FAILED");
+            console.error("==> Test was supposed to fail but all items were found...");
+        } else if (error_text.length !== 0 && should_fail === false) {
```

Comment:
Any particular reason you went with a new `should_fail_happy` variable instead of reusing `error_text` like the regular output does? Just the early return?